vendor/itemproperties.lua

```diff
@@ -161,6 +160,7 @@ function Addon:GetItemProperties(arg1, arg2)
     item.IsUnsellable = not item.UnitValue or item.UnitValue == 0
     item.ExpansionPackId = getItemInfo[15]  -- May be useful for a rule to vendor previous ex-pac items, but doesn't seem consistently populated
     item.IsAzeriteItem = (getItemInfo[15] == 7) and C_AzeriteEmpoweredItem.IsAzeriteEmpoweredItemByID(item.Link);
+    item.IsCraftingReagent = getItemInfo[17]
```

Comment:
When was the 17th item added? Does it exist in Classic? I don't know off hand, but its something we need to know.

Reply:
It was added in Legion and good point about the classic version. I don't have it installed so can't actually check but I would guess not since nothing in classic is marked with "Crafting Reagent" in the tooltip.